app/src/main/java/de/ph1b/audiobook/features/BookAdder.kt

```diff
@@ -7,11 +7,7 @@ import androidx.core.content.ContextCompat
 import de.paulwoitaschek.chapterreader.ChapterReader
 import de.ph1b.audiobook.common.comparator.NaturalOrderComparator
 import de.ph1b.audiobook.common.sparseArray.emptySparseArray
-import de.ph1b.audiobook.data.Book
-import de.ph1b.audiobook.data.BookContent
-import de.ph1b.audiobook.data.BookMetaData
-import de.ph1b.audiobook.data.BookSettings
-import de.ph1b.audiobook.data.Chapter
+import de.ph1b.audiobook.data.*
```

Comment:
No wildcard imports

Reply:
Gotcha. This is Android Studio's default Kotlin style settings.

Reply:
* Use `set from` and then `prefdefined style ` and then `kotlin style guide`.
* Use 2 space intents.
* No wildcards at all: https://android.github.io/kotlin-guides/style.html#import-statements

I heard they are working on adding a set for the android styleguide.

![image](https://user-images.githubusercontent.com/743462/46921295-fa6c2d80-cff9-11e8-9046-3a23b61d0892.png)

Reply:
Thanks! I knew about the imports setting but not about the `set from` features.